extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -398,12 +398,17 @@ public function getCommunityHeaderBackgroundUrl(): string {
 		$thumbnailUrl = '';
 		$originalUrl = $this->getSettings()['community-header-background-image'];
 
-		if ( VignetteRequest::isVignetteUrl( $originalUrl ) ) {
-			$thumbnailUrl = VignetteRequest::fromUrl( $originalUrl )
-				->zoomCrop()
-				->width( self::COMMUNITY_HEADER_BACKGROUND_WIDTH )
-				->height( self::COMMUNITY_HEADER_BACKGROUND_HEIGHT )
-				->url();
+		// @todo fix the issue with vignette urls without /latest and remove try-catch
+		try {
+			if ( VignetteRequest::isVignetteUrl( $originalUrl ) ) {
+				$thumbnailUrl = VignetteRequest::fromUrl( $originalUrl )
+					->zoomCrop()
+					->width( self::COMMUNITY_HEADER_BACKGROUND_WIDTH )
+					->height( self::COMMUNITY_HEADER_BACKGROUND_HEIGHT )
+					->url();
+			}
+		} catch (Exception $e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS108)